src/main/scala/nl/knaw/dans/bag/util/constants.scala

```diff
@@ -0,0 +1,116 @@
+package nl.knaw.dans.bag.util
+
+
+/**
+ * as used in files.xml
+ */
+object FileAccessCategory extends Enumeration {
+  type FileAccessCategory = Value
+
+  val ANONYMOUS: FileAccessCategory = Value
+  val RESTRICTED_GROUP: FileAccessCategory = Value
+  val RESTRICTED_REQUEST: FileAccessCategory = Value
+  val KNOWN: FileAccessCategory = Value
+  val NONE: FileAccessCategory = Value
+}
+
+/**
+ * as used in dataset.xml
+ */
+object DatasetAccessCategory extends Enumeration {
+  type DatasetAccessCategory = Value
+
+  val OPEN_ACCESS: DatasetAccessCategory = Value
+  val OPEN_ACCESS_FOR_REGISTERED_USERS: DatasetAccessCategory = Value
+  val GROUP_ACCESS: DatasetAccessCategory = Value
+  val REQUEST_PERMISSION: DatasetAccessCategory = Value
+  val NO_ACCESS: DatasetAccessCategory = Value
+}
+
+//subset of the ISO 639-2 language codes
+object XmlLang extends Enumeration {
+  type XmlLang = Value
+
+  val ENG: XmlLang = Value
+  val NLD: XmlLang = Value
+  val FRA: XmlLang = Value
+  val DEU: XmlLang = Value
+}
+
+object DcTermsElementType extends Enumeration {
+  type DcTermsElementType = Value
+
+  //dc terms:
+  val ABSTRACT: DcTermsElementType = Value
+  val ACCESS_RIGHTS: DcTermsElementType = Value("accessRights")
+  val ACCRUAL_METHOD: DcTermsElementType = Value("accrualMethod")
+  val ACCRUAL_PERIODICITY: DcTermsElementType = Value("accrualPeriodicity")
+  val ACCRUAL_POLICY: DcTermsElementType = Value("accrualPolicy")
+  val ALTERNATIVE: DcTermsElementType = Value
+  val AUDIENCE: DcTermsElementType = Value
+  val AVAILABLE: DcTermsElementType = Value
+  val BIBLIOGRAPHIC_CITATION: DcTermsElementType = Value("bibliographicCitation")
+  val CONFORMS_TO: DcTermsElementType = Value("conformsTo")
+  //  val CONTRIBUTOR: DcTermsElementType = Value
+  //  val COVERAGE: DcTermsElementType = Value
+  val CREATED: DcTermsElementType = Value
+  //  val CREATOR: DcTermsElementType = Value
+  //  val DATE: DcTermsElementType = Value
+  val DATE_ACCEPTED: DcTermsElementType = Value("dateAccepted")
+  val DATE_COPYRIGHTED: DcTermsElementType = Value("dateCopyrighted")
+  val DATE_SUBMITTED: DcTermsElementType = Value("dateSubmitted")
```

Comment:
Not even considering the overall approach, this is a too big unstructured pile of types.
How to reuse (for example) dates by the [deposit-api](https://github.com/DANS-KNAW/easy-deposit-api/blob/39bfa9fda60354640146f6a517162246a4f470f3/src/main/scala/nl.knaw.dans.easy.deposit/docs/dm/Date.scala#L24-L35) other enums in the same dm package of the api, relation types and whatever?